lib/rubies/game.rb

```diff
@@ -147,8 +147,18 @@ def check_answer(current, input, target)
         routine = lambda { eval(input) }
         output = routine.call
         puts "=> #{output}"
-        puts
-        output == target
+        
+	# Ensure they don't just paste in the required output
+	trimmedInput = input.to_s.delete '\'\"'
+	trimmedTarget = target.to_s.delete '\'\"'
+	if (trimmedInput != trimmedTarget) then
```

Comment:
Inconsistent indentation detected.<br>Do not use `then` for multi-line `if`.<br>Tab detected.